tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -82,6 +85,31 @@ Verify NFS Volume Already Created
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: A volume named ${containerVolName} already exists. Choose a different volume name.
 
+Reboot VM and Verify Basic VCH Info
+    Log To Console  Rebooting VCH\n - %{VCH-NAME}
+    Reboot VM  %{VCH-NAME}
+
+    Log To Console  Getting VCH IP ...
+    ${new_vch_ip}=  Get VM IP  %{VCH-NAME}
+    Log To Console  New VCH IP is ${new_vch_ip}
+    Replace String  %{VCH-PARAMS}  %{VCH-IP}  ${new_vch_ip}
```

Comment:
@mhagen-vmware 10-01-VCH-Restart.robot line 66 and 1-12-Docker-RMI.robot line 33 use `Replace String` w/o having the result being copied back to the env var since the keyword itself should handle that.
I've added a couple of checks after the `Replace String` keyword in this test case just to double check that the new IP is present in the VCH_PARAMS.  I also added an check to make sure the result of `Replace String` is exactly what `VCH_PARAMS` should be with the new vch ip.

Let me know if these checks are ok with you.  Also, let me know if you want me to go and change those other two files using `Replace String` this way.